components/dialog/dialog-mixin.js

```diff
@@ -270,7 +270,7 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 		const availableHeight = this._ifrauContextInfo
 			? this._ifrauContextInfo.availableHeight - this._margin.top - this._margin.bottom
 			: window.innerHeight - this._margin.top - this._margin.bottom;
-		let preferredHeight = 2;
+		let preferredHeight = 3; // content height is off by 2px and we need an extra 1px to account for rounding when zoomed
```

Comment:
From my measurements, the existing `2` value is here because the measurement of the content area tends to underestimate by `2px`. I'm not totally sure why, but when I compare the measurements it gets here to what I see in dev tools, it's off by 2 consistently.

I'm not sure why it's using `offsetHeight` for the content but `getBoundingClientRect().height` for the header/footer, but it's been doing it that way forever so I'm hesitant to change it.